src/rpc/register.h

```diff
@@ -10,23 +10,23 @@
 class CRPCTable;
 
 /** Register block chain RPC commands */
-void RegisterBlockchainRPCCommands(CRPCTable &tableRPC);
+void RegisterBlockchainRPCCommands(CRPCTable &t);
```

Comment:
Agree with sipa: the real solution is to eliminate the global variable, not to use a less descriptive name here.